test/test_configs.py

```diff
@@ -516,6 +517,151 @@ def test_exploration(self, network, wrapper):
+                MLP(num_cells=[10], out_features=2 * env.action_spec.shape[-1])
+            ),
+            in_keys=["observation_vector"],
+            out_keys=["loc", "scale"],
+        )
+        policy_operator = ProbabilisticActor(
+            module=td_module,
+            dist_param_keys=["loc", "scale"],
+            spec=env.action_spec,
+            distribution_class="TanhNormal",
+        )
+        value_operator = ValueOperator(
+            MLP(num_cells=[10], out_features=1), in_keys=["observation_vector"]
+        )
+        return policy_operator, value_operator
+
+    def _make_model_sac(self, env):
+        td_module = TensorDictModule(
+            module=NormalParamWrapper(
+                MLP(num_cells=[10], out_features=2 * env.action_spec.shape[-1])
+            ),
+            in_keys=["observation_vector"],
+            out_keys=["loc", "scale"],
+        )
+        policy_operator = ProbabilisticActor(
+            module=td_module,
+            dist_param_keys=["loc", "scale"],
+            spec=env.action_spec,
+            distribution_class="TanhNormal",
+        )
+        qvalue_operator = ValueOperator(
+            MLP(num_cells=[10], out_features=1),
+            in_keys=["action", "observation_vector"],
+        )
+        value_operator = ValueOperator(
+            MLP(num_cells=[10], out_features=1), in_keys=["observation_vector"]
+        )
+        return policy_operator, qvalue_operator, value_operator
+
+    def _make_model_ddpg(self, env):
+        td_module = TensorDictModule(
+            module=DdpgMlpActor(
+                mlp_net_kwargs={"num_cells": 10}, out_features=env.action_spec.shape[-1]
+            ),
+            in_keys=["observation_vector"],
+            out_keys=["param"],
+        )
+        policy_operator = ProbabilisticActor(
+            module=td_module,
+            dist_param_keys=["param"],
+            spec=env.action_spec,
+            distribution_class="TanhDelta",
+        )
+        qvalue_operator = ValueOperator(
+            DdpgMlpQNet(), in_keys=["observation_vector", "action"]
+        )
+        return policy_operator, qvalue_operator
+
+    def _make_model_redq(self, env):
+        td_module = TensorDictModule(
+            module=NormalParamWrapper(
+                MLP(num_cells=[10], out_features=2 * env.action_spec.shape[-1])
+            ),
+            in_keys=["observation_vector"],
+            out_keys=["loc", "scale"],
+        )
+        policy_operator = ProbabilisticActor(
+            module=td_module,
+            dist_param_keys=["loc", "scale"],
+            spec=env.action_spec,
+            distribution_class="TanhNormal",
+        )
+        qvalue_operator = ValueOperator(
+            MLP(num_cells=[10], out_features=1),
+            in_keys=["action", "observation_vector"],
+        )
+        return policy_operator, qvalue_operator
+
+    @pytest.mark.parametrize("model", ["ddpg", "dqn", "ppo", "redq", "sac"])
```

Comment:
this is a bit too general
we should have one function per loss, and test the hyperparams of each of them independently